generators/client/templates/src/components/authentication/SignOutButton.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import { Redirect } from "react-router-dom";
+
+const SignOutButton = () => {
+  const [shouldRedirect, setShouldRedirect] = useState(false);
+  const signOut = (event) => {
+    event.preventDefault();
+    fetch("/api/v1/user-sessions/current", {
```

Comment:
i believe this path is incorrect.
- clicking on the button issues a DELETE but does not sign out the user or redirect the page
- the correct route would `"/api/v1/user-sessions` based on the router, otherwise we could also update the router